internal/session/repository_session_test.go

```diff
@@ -107,6 +116,9 @@ func TestRepository_ListSession(t *testing.T) {
 				s := TestSession(t, conn, wrapper, composedOf)
 				_ = TestState(t, conn, s.PublicId, StatusActive)
 				testSessions = append(testSessions, s)
+				for i := 0; i < tt.withConnections; i++ {
+					_ = TestConnection(t, conn, s.PublicId, "127.0.0.1", 22, "127.0.0.1", 22)
```

Comment:
Actually, after reading through more of the code. Since `States` are not included in `TestSession` and aren't included in `ComposedOf` I think it makes sense to leave Connections separate, as a thing that is created outside of the session.

Reply:
Sounds reasonable.